src/fu-tool.c

```diff
@@ -2018,10 +2019,14 @@ fu_util_security (FuUtilPrivate *priv, gchar **values, GError **error)
 	g_print ("%s \033[1m%s\033[0m\n", _("Host Security ID:"),
 		 fu_engine_get_host_security_id (priv->engine));
 
+	/* show or hide different elements */
+	if (priv->show_all_devices)
```

Comment:
this feels like it's overloading `--show-all-devices`.  Either it should be a new option `--show-all-security`, or I think we should rename to `--show-all`.

Reply:
> or I think we should rename to --show-all

I think you're right. New patch in 5..4...